source/infrastructure/dev-tools/index.html.md.erb

```diff
@@ -9,6 +9,9 @@ weight: 10
 
 Installation of each product should be carried out following the distributors current guidance, depending on your operating system.
 
+## Desktop Docker
+The Desktop docker application interface is a cross-platform GUI tool to help developers build images to create containers for application development and deployment to cloud containers in azure. Desktop docker allows the user full administrative rights over all aspects of the development of applications run in a container. For tech guidance [instructions](/infrastructure/docker).
+
```

Comment:
It may be worth mentioning that while Docker engine runs on Linux, Docker desktop runs it transparently in a virtual machine on Mac and Windows